install/set-up-and-migrate-database.sh

```diff
@@ -8,8 +8,14 @@ until $dc exec postgres psql -U postgres -c "select 1" >/dev/null 2>&1 || [ $RET
   echo "Waiting for postgres server, $((RETRIES--)) remaining attempts..."
   sleep 1
 done
-$dc exec postgres psql -qAt -U postgres -c "ALTER TABLE IF EXISTS sentry_groupedmessage DROP CONSTRAINT IF EXISTS sentry_groupedmessage_project_id_id_515aaa7e_uniq;"
-$dc exec postgres psql -qAt -U postgres -c "DROP INDEX IF EXISTS sentry_groupedmessage_project_id_id_515aaa7e_uniq;"
+
+$dcr web shell -c "
+from django.db import connection
+
+with connection.cursor() as cursor:
+  cursor.execute('ALTER TABLE IF EXISTS sentry_groupedmessage DROP CONSTRAINT IF EXISTS sentry_groupedmessage_project_id_id_515aaa7e_uniq;')
+  cursor.execute('DROP INDEX IF EXISTS sentry_groupedmessage_project_id_id_515aaa7e_uniq;')
+"
```

Comment:
Want to make a comment about why we're doing it this way so we remember to keep this use case in mind in the future?

Reply:
done